src/routes/PublicLayoutRouter/PublicLayoutRouter.js

```diff
@@ -28,9 +28,10 @@ export default function PublicLayoutRouter (props) {
         <Route path={`/${POST_DETAIL_MATCH}`} exact component={PublicPostDetail} />
         <Route path='/:context(groups)/:groupSlug' exact component={PublicGroupDetail} />
         <Route path='/:context(public)/:view(map)' component={MapExplorerLayoutRouter} />
-        <Route path='/:context(public)/:view(groups)' exact component={GroupExplorerLayoutRouter} />
+        <Route path='/:context(public)/:view(groups)(.*)' exact component={GroupExplorerLayoutRouter} />
         {/* Remove this once we show the public stream */}
         <Redirect exact from={`/public/${POST_DETAIL_MATCH}`} to='/post/:postId' />
+        <Redirect exact from={`/:context(groups)/:groupSlug/:view(events|groups|map|members|projects|settings|stream|topics|custom)/${POST_DETAIL_MATCH}`} to='/post/:postId' />
```

Comment:
It may be a small rabbithole, but one I want us to keep an eye one as this I think is the case: This I think can be just `(.*)/${POST_DETAIL_MATCH}` and not break anything. Try it if you don't mind, and I am sorry ahead of time if I've missed something.

Reply:
won't this make it not possible to view posts from the public map though? /public/map/post/:id will redirect to /post/:id